crates/bitwarden-uniffi/src/lib.rs

```diff
@@ -107,6 +107,8 @@ fn init_logger() {
 
     #[cfg(target_os = "android")]
     android_logger::init_once(
-        android_logger::Config::default().with_max_level(uniffi::deps::log::LevelFilter::Info),
+        android_logger::Config::default()
+            .with_tag("com.bitwarden.sdk")
```

Comment:
Added this to make sure all the SDK logs consistently use the same tag, for easier filtering. Not necessary for this fix